galaxykit/collections.py

```diff
@@ -176,16 +177,16 @@ def move_collection(
     return True
 
 
-def delete_collection(client, namespace, collection, version):
+def delete_collection(client, namespace, collection, version = 'None', repository = 'published'):
     """
     Delete collection version
     """
-
-    if version == None:
-        delete_url = f"v3/plugin/ansible/content/published/collections/index/{namespace}/{collection}/"
+    if version == 'None':
```

Comment:
let's not introduce special strings, what's the problem with `None`?

Reply:
Do you know how to pass None as argument in command? This will be passed as string.

Reply:
https://github.com/ansible/galaxykit/pull/25#discussion_r870318280